firedrake/assemble_expressions.py

```diff
@@ -457,12 +457,13 @@ def __exit__(self, *args, **kwargs):
 
 @PETSc.Log.EventDecorator()
 @known_pyop2_safe
-def evaluate_expression(expr, subset=None):
+def evaluate_expression(expr, subset=None, dat_map=None):
```

Comment:
What is this for?

Reply:
That was relevant to a different part of my project, I'll remove it here. Thanks!